docker/openemr/flex-3.10/autoconfig.sh

```diff
@@ -303,3 +303,7 @@ if [ "$REDIS_SERVER" != "" ] &&
     # Ensure only configure this one time
     touch /etc/php-redis-configured
 fi
+
+if [ "TURN_API_ON" == "yes" ]; then
+    mysql -u openemr --password="openemr" -h mysql -e "UPDATE globals SET gl_value = 1 WHERE gl_name = \"rest_api\"" openemr
```

Comment:
Overall command looks good, but will need to include host. Note that the mysql host (`MYSQL_HOST`), openemr mysql user (`MYSQL_USER`), openemr mysql password (`MYSQL_PASS`), and  openemr database (`MYSQL_DATABASE`) can be set as docker environment variables. The MYSQL_HOST is mandatory. And the others are optional (if they are note set, then they will use the settings that you used in above command (so, need to check if the environment variable exists; use it if it exists; use default setting if it doesn't exist).

Reply:
This should be done at the end of the auto_setup() function

Reply:
Hello @bradymiller, seen. Working on this